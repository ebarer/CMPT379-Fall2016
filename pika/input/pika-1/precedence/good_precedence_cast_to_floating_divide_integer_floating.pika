exec {

  print [ 41 | float ] / 3.0, _n_.
  print [ 62 | float ] / 5.9, _n_.
  print [ 18 | float ] / 4.3, _n_.
  print [ 10 | float ] / 7.4, _n_.
  print [ 2 | float ] / 7.0, _n_.
  print [ 82 | float ] / 8.6, _n_.
  print [ 45 | float ] / 4.2, _n_.
  print [ 40 | float ] / 1.1, _n_.
  print [ 81 | float ] / 5.3, _n_.
  print [ 56 | float ] / 6.1, _n_.
  print [ 8 | float ] / 6.1, _n_.
  print [ 75 | float ] / 6.2, _n_.

  print 4.0 / [ 50 | float ], _n_.
  print 2.9 / [ 35 | float ], _n_.
  print 9.0 / [ 15 | float ], _n_.
  print 1.8 / [ 66 | float ], _n_.
  print 2.6 / [ 57 | float ], _n_.
  print 4.7 / [ 6 | float ], _n_.
  print 5.9 / [ 64 | float ], _n_.
  print 4.2 / [ 9 | float ], _n_.
  print 5.6 / [ 32 | float ], _n_.
  print 1.7 / [ 97 | float ], _n_.
  print 9.3 / [ 30 | float ], _n_.
  print 8.8 / [ 96 | float ], _n_.

  print [ 95 | float ] / [ 94 | float ], _n_.
  print [ 73 | float ] / [ 87 | float ], _n_.
  print [ 68 | float ] / [ 24 | float ], _n_.
  print [ 94 | float ] / [ 46 | float ], _n_.
  print [ 75 | float ] / [ 10 | float ], _n_.
  print [ 18 | float ] / [ 13 | float ], _n_.
  print [ 9 | float ] / [ 62 | float ], _n_.
  print [ 12 | float ] / [ 96 | float ], _n_.
  print [ 25 | float ] / [ 67 | float ], _n_.
  print [ 8 | float ] / [ 59 | float ], _n_.
  print [ 15 | float ] / [ 25 | float ], _n_.
  print [ 14 | float ] / [ 24 | float ], _n_.

}

