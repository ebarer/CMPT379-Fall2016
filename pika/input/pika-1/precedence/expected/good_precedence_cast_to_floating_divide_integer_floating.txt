13.6667
10.5085
4.18605
1.35135
0.285714
9.53488
10.7143
36.3636
15.283
9.18033
1.31148
12.0968
0.08
0.0828571
0.6
0.0272727
0.045614
0.783333
0.0921875
0.466667
0.175
0.0175258
0.31
0.0916667
1.01064
0.83908
2.83333
2.04348
7.5
1.38462
0.145161
0.125
0.373134
0.135593
0.6
0.583333
